Feature-1/one-more-file.txt

```diff
@@ -0,0 +1,3 @@
+this file was also added in feature1 folder
```

Comment:
its nice you have added this line in the file.

Reply:
Thanks for this comment